fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_coffin.dm

```diff
@@ -40,17 +40,17 @@
 
 /obj/structure/closet/crate/coffin/securecoffin
 	name = "secure coffin"
-	desc = "For those too scared of having their place of rest disturbed."
+	desc = "For insecure suckers to safely slumber in their secure sleeping space."
```

Comment:
this line is cringe sorry

Reply:
Shameful the stupid senior supervisors sense of humour sucks